docs/source/data.rst

```diff
@@ -27,14 +27,14 @@ This module defines standard data structure for pattern commands.
 .. autoclass:: MeasureUpdate
 
 
-:mod:`graphix.pauli` module
-+++++++++++++++++++++++++++
+:mod:`graphix.fundamentals` module
+++++++++++++++++++++++++++++++++++
 
-This module defines standard data structure for Pauli operators, measurement planes and their transformations.
+This module defines fundamental components of quantum computing.
```

Comment:
too much generalization I think.

Reply:
Actually "measurement planes and their transformations" part was moved to other files.

How about this?

```suggestion
This module defines standard data structure for Pauli operators.
```

Reply:
thanks, yes looks good.